### PR TITLE
add Example applications section

### PR DESCRIPTION
**Pull request by kamil-kielczewski**

### CODE REVIEW

**Comment by kamil-kielczewski** [comment]
@abhisheksoni27 I change "Example applications" section to "Demos" and make commit

**Comment by Somebi** [comment]
Cool demo.

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ Or alternatively you can experiment around with the [kernel playground here](htt
 * [Contributing](#contributing)
 * [Terms Explained](#terms-explained)
 * [License](#license)
+* [Demos](#example-applications)
```
Comment (HarshKhandeparkar) [inline README.md]: The link should be #demos

Comment (kamil-kielczewski) [reply README.md]: thanks - now is fixed

**Comment by robertleeplummerjr** [comment]
An "Examples" area was recently added that took this change into account.  Ty very much for adding it!

**Pull request closed without merging**